rfcs/proposed/task_group_dynamic_dependencies/README.md

```diff
@@ -196,70 +228,12 @@ by a dependence on the current task itself, so we can ensure that we can safely
 update the incoming dependencies for those tasks without worrying about
 potential race conditions. 
 
-One possible spelling for this function would be `transfer_successors_to(h)`, 
+One possible spelling for this function would be `tbb::transfer_successors_to(h)`, 
```

Comment:
I would not be that specific for now; providing this function in the `task_group` class might make more sense.

Reply:
Reverted